docker/Makefile.docker

```diff
@@ -1,30 +1,40 @@
 export DOCKER_BUILDKIT=1
 
+export AR_REQS_REPO ?= ${AR_REPO_PREFIX}/umbrella-reqs
+export CI_REQS_REPO ?= codecov/umbrella-reqs
+
+# `LC_ALL=C` is added to `sort` to ensure you get the same order across systems.
+# Otherwise, a Mac may sort according to en_US.UTF-8 while CI may sort according to C/POSIX.
```

Comment:
lol, I hate this behavior so so much. why are these coreutils just fundamentally "non-portable" in such a way :-(